requirements.txt

```diff
@@ -0,0 +1 @@
+ethereum==1.1.0
```

Comment:
for some reason I always think requests is part of the standard library, adding it now.